src/renderer/components/CurrencyBadge.js

```diff
@@ -95,7 +95,12 @@ function CurrencyBadge({ currency }: { currency: CryptoCurrency | TokenCurrency
         >
           {currency.ticker}
         </Box>
-        <Box ff="Inter" color="palette.text.shade100" fontSize={5} data-e2e="currencyBadge">
+        <Box
+          ff="Inter"
+          color="palette.text.shade100"
+          fontSize={5}
+          data-automation-id="currency-badge"
```

Comment:
I recommend to just use `id` and `className`s (classes)

- you benefit the fact react/ the DOM will warn you when there are duplicated ids
- it is a generic concept and valid for the purpose of identifying **unique** elements
- it's easier to select because you can just `#id`

Reply:
I agree with you, I didn't want to overlap potential dev ids, search & replace incoming then 👍.

Don't mind about this PR for the moment, I'm migrating those commits from lld-v2.